### PR TITLE
Bind static VipsImageImpl.black to vips_black

### PR DESCRIPTION
**Pull request by stuart12**
Add new binding to create a black image.

### CODE REVIEW

File: src/main/java/com/criteo/vips/VipsImageImpl.java [hunk 1]
```diff
@@ -57,6 +57,21 @@ public VipsImageImpl(VipsImageImpl image, PixelPacket color) throws VipsExceptio
...
+        blackNative(width, height);
+    }
+    private native void blackNative(int width, int height) throws VipsException;
+
+    /** Make a black unsigned char image of a specified size
+     *
+     * @param width output width
+     * @param height output height
+     * @throws VipsException
+     */
```
Comment (dbouron) [inline src/main/java/com/criteo/vips/VipsImageImpl.java]: Please declare and move comment  it in VipsImage.java

File: src/main/java/com/criteo/vips/VipsImageImpl.java [hunk 2]
```diff
@@ -57,6 +57,21 @@ public VipsImageImpl(VipsImageImpl image, PixelPacket color) throws VipsExceptio
         newFromImage(image, color);
     }
 
+    private VipsImageImpl(int width, int height) throws VipsException {
+        blackNative(width, height);
+    }
```
Comment (dbouron) [inline src/main/java/com/criteo/vips/VipsImageImpl.java]: maybe we can just declare a static method

File: src/main/java/com/criteo/vips/VipsImageImpl.java (hunk 1 above)
Comment (stuart12) [reply src/main/java/com/criteo/vips/VipsImageImpl.java]: you cannot put code in an interface

File: src/main/java/com/criteo/vips/VipsImageImpl.java (hunk 2 above)
Comment (stuart12) [reply src/main/java/com/criteo/vips/VipsImageImpl.java]: done

**Pull request merged**